receiver/kymastatsreceiver/kyma_scraper.go

```diff
@@ -134,24 +134,33 @@ func (ks *kymaScraper) unstructuredToStats(module unstructured.Unstructured) (*m
 		return nil, &fieldNotFoundError{"state"}
 	}
 
-	unstructuredConds, found, err := unstructured.NestedSlice(status, "conditions")
-	if err != nil {
-		return nil, err
-	}
-	if !found {
-		return nil, &fieldNotFoundError{"conditions"}
-	}
-
 	stats := &moduleStats{
 		state:     state,
 		namespace: module.GetNamespace(),
 		kind:      module.GetKind(),
 	}
 
+	unstructuredConds, found, err := unstructured.NestedSlice(status, "conditions")
+	if err != nil || !found {
+		ks.logger.Error("Error converting unstructured module to stats, no module condition found",
+			zap.Error(err),
+			zap.String("name", module.GetName()),
+			zap.String("namespace", module.GetNamespace()),
+			zap.String("kind", module.GetKind()),
+		)
+		return stats, nil
+	}
```

Comment:
The error log is misleading: `found` being false signalizes that conditions are not found, and a non-nil `err` signalizes that conditions are found, but the type is wrong. Let us maybe have 2 different logs with level warning, since it's not a real error:
```suggestion
	unstructuredConds, found, err := unstructured.NestedSlice(status, "conditions")
	if err != nil {
		ks.logger.Warn("Failed to retrieve conditions: conditions are not a slice",
			zap.Error(err),
			zap.String("name", module.GetName()),
			zap.String("namespace", module.GetNamespace()),
			zap.String("kind", module.GetKind()),
		)
		return stats, nil
	}
	if !found {
			ks.logger.Warn("Failed to retrieve conditions: conditions not found",
			zap.Error(err),
			zap.String("name", module.GetName()),
			zap.String("namespace", module.GetNamespace()),
			zap.String("kind", module.GetKind()),
		)
		return stats, nil
	}
```